walletlibrary/src/main/java/com/microsoft/walletlibrary/wrapper/VerifiedIdRequester.kt

```diff
@@ -22,18 +24,54 @@ object VerifiedIdRequester {
     // sends the issuance response to VC SDK and returns a VerifiedId if successful.
     internal suspend fun sendIssuanceResponse(
         issuanceRequest: IssuanceRequest,
-        requirement: Requirement
+        requirement: Requirement,
+        requestState: String? = null,
+        issuanceCallbackUrl: String? = null
     ): VerifiedId {
         val issuanceResponse = IssuanceResponse(issuanceRequest)
         issuanceResponse.addRequirements(requirement)
         when (val result = VerifiableCredentialSdk.issuanceService.sendResponse(issuanceResponse)) {
-            is Result.Success -> return VerifiableCredential(result.payload, issuanceRequest.contract)
+            is Result.Success -> {
+                val issuanceCompletionResponse = requestState?.let {
```

Comment:
I guess here. If the issuanceCompletion call returns back an error, will the developer get back in error in the outer layer or the issued VerifedId?

Reply:
I changed how failure is handled during callback. They are only logged on failure. The only result returned is the actual flow's.